infoview/info.tsx

```diff
@@ -75,7 +74,7 @@ export function Info(props: InfoProps) {
                         </div> }
                 </div>
                 <div>
-                    <Widget widget={widget} fileName={loc.file_name} onEdit={onEdit} />
+                    <Widget widget={widget} fileName={loc.file_name} />
```

Comment:
You should merge master.  This was already changed there.

Reply:
eek sorry I thought I did merge master!